### ISSUE TITLE
[9.5 RC1] New Install Fails during Database Creation

### ISSUE THREAD

**Issue by brwainer** [glpi-project/glpi#7425]
**Describe the bug**

During Database creation in the wizard, I receive a 504 Gateway Timeout. 

**Page(s) URL**

http://192.168.129.15/glpi/install/install.php

**To reproduce**

Essentially, try to install GLPI from scratch on CentOS 8.1. For completeness, I checked my command history and copied everything here.

1. Install new OS from CentOS 8.1.1911 ISO (select category "Minimal Install", options "Guest Agents", "Standard", "Headless Management", "System Tools")
2. Install repositories:
```
dnf update -y
reboot
dnf install -y epel-release yum-utils
dnf config-manager --set-enabled PowerTools
dnf install -y https://rpms.remirepo.net/enterprise/remi-release-8.rpm
```
3. Install Software and set settings:
```
dnf install -y httpd mariadb-server mariadb php php-gd php-mysql php-mcrypt php-apcu php-xmlrpc php-pecl-zendopcache php-ldap php-imap php-mbstring php-simplexml php-xml php-pecl-zip php-intl
nano /etc/httpd/conf/httpd.conf //change AllowOverride to All for /var/www/html/
nano /etc/php.ini //change max_execution_time to 600
systemctl enable --now mariadb
mysql_secure_installation //set root password
firewall-cmd --add-service=http --permanent
firewall-cmd --reload
```
4. Download GLPI and set permissions:
```
cd /var/www/html/
wget https://github.com/glpi-project/glpi/releases/download/9.5.0-rc1/glpi-9.5.0-rc1.tgz
tar xvf /var/www/html/glpi-9.5.0-rc1.tgz
chmod 755 /var/www/html/glpi -R
chown apache:apache /var/www/html/glpi -R
semanage fcontext -a -t httpd_sys_rw_content_t "/var/www/html/glpi/files(/.*)?"
restorecon -R -v /var/www/html/glpi/files
semanage fcontext -a -t httpd_sys_rw_content_t "/var/www/html/glpi/config(/.*)?"
restorecon -R -v /var/www/html/glpi/config
semanage fcontext -a -t httpd_sys_rw_content_t "/var/www/html/glpi/marketplace(/.*)?"
restorecon -R -v /var/www/html/glpi/marketplace
setsebool httpd_can_network_connect=on
setsebool httpd_can_network_connect_db=on
setsebool httpd_can_sendmail=on
```
5. Start Apache
`systemctl enable --now httpd`
6. Open browser to (ip)/glpi/ and follow installer. Note that all items are checked green except for the CAS extensions.
7. After pressing "Continue" on the database creation screen, you will get a 504 Gateway Timeout after about 45-60 seconds.
8. If you then go to (ip)/glpi/ the login screen loads, and it appears everything is OK, but there can be no confidence that the installation completed 100%.

**Expected behavior**

The installer to not have a timeout error.

**Logs**

In /var/log/httpd/error_log it logs:
`[Sun Jun 14 21:57:40.438033 2020] [proxy_fcgi:error] [pid 8961:tid 140669431572224] (70007)The timeout specified has expired: [client 192.168.128.3:46406] AH01075: Error dispatching request to : (polling), referer: http://192.168.129.15/glpi/install/install.php`

glpi/files/_log/php-errors.log has:
```
[2020-06-15 01:56:22] glpiphplog.WARNING: Test logger  
[2020-06-15 01:56:40] glpiphplog.NOTICE: GLPIKey::getLegacyKey() in /var/www/html/glpi/inc/glpikey.class.php line 116
Called method is deprecated  {"user":"@itam01.ipa.wainer-it.com"}
```
There are no other error logs in that folder. I also checked all other Apache, PHP, and MariaDB logs and did not find anything related.

**Screenshots**

Not Applicable

**Your GLPI setup (you can find it in Setup > General menu, System tab)**

```
Not Applicable
```

**Additional context**

I also performed the same steps using the 9.4.6 release and had the same problem. In between attempts I dropped the created table and removed the glpi folder.

Additionally I reviewed the instructions at https://glpi-install.readthedocs.io/en/latest/prerequisites.html and have these notes:

- php-intl and php-zip are new dependencies for 9.5 compared to 9.4.6. php-intl in particular is difficult to know about, because the install.php page comes up completely blank with no error message if this is not installed.
- several of the php configuration items called out are no longer applicable - especially register_globals which was removed in PHP 5.4

**Comment by brwainer** [comment]
It appears that because I am not a contributor to this project, I cannot apply the requested label.

**Comment by brwainer** [comment]
Adding "Timeout 600" to "nano /etc/httpd/conf/httpd.conf" allowed the database step to complete. Either the database step needs to be improved to <60 seconds, do this change needs to be added to your prerequisites.

**Comment by cedric-anne** [comment]
Hi,

Indeed, installation process using web UI may timeout on this kind of configuration (php call through fcgi). Reworking installation UI to prevent this issue will not be done on GLPI 9.5 (too critical to start the job now).

I recommand you to use the CLI command to install GLPI. Call `bin/console db:install --help` on GLPI directory for more information. Using this command will generate files and directories that must be writable by your webserver user, so be careful to run this command with the most suitable user.

Regards

**Comment by trasher** [comment]
For the cli, see also https://glpi-install.readthedocs.io/en/latest/command-line.html

**Comment by cedric-anne** [comment]
> Indeed, installation process using web UI may timeout on this kind of configuration (php call through fcgi). Reworking installation UI to prevent this issue will not be done on GLPI 9.5 (too critical to start the job now).

@brwainer  Could you test with #7434 ? I am not sure it fixes the problem, but if it does, it may be included in GLPI 9.5 as it is a small fix.
